tests/main.py

```diff
@@ -43,6 +43,8 @@ def setup_module():
     file = Gio.File.new_for_path(repo_path)
     repo = OSTree.Repo.new(file)
     repo.create(OSTree.RepoMode.BARE, None)
+    remote_path = os.path.join(os.getcwd(), "tests/ostree/repo")
+    repo.remote_add("flathub", f"file://{remote_path}")
```

Comment:
Hey Bart~ what's the difference between this and what we are doing a few lines above:
```
    installation_path = os.path.join(workspace.name, "flatpak")
    repo_path = os.path.join(installation_path, "repo")

    os.mkdir(installation_path)
    os.environ["FLATPAK_USER_DIR"] = installation_path
```

Reply:
One of the commits adds hand-crafted ostree repo which is here added as remote. This way ostree-commit-based "last updated" dates are stable; for example it just broke for `v1` too because WPS Office has been updated.

Reply:
(This is the "it worked by coincidence" thing I mentioned previously.)

Reply:
Got it :+1: